site/content/en/docs/Reference/fleet.md

```diff
@@ -163,6 +163,15 @@ spec:
       version: "" # empty an existing label value
     annotations:
       otherkey: setthisvalue
+  #
+  # [Stage:Alpha]
```

Comment:
Nice!

Please also add a section below "The `spec` field is the actual `Fleet` specification and it is composed as follow:" which is where we also explain part of the new data structure. it's basically a repeat of above in a slightly different format.

Also same with each of the other `Reference/*.md` files in this PR.

Reply:
Sorry - that wasn't what I meant.

I'm talking about this section here: 

![image](https://github.com/googleforgames/agones/assets/298370/6e3f9267-3181-40c7-84eb-da3ee4af492c)

This is where we also explain what each part of the resource (Fleet, GameServer, etc) configuration does, and extra details about that.

Does that make more sense?

Reply:
So to be extra clear - revert this commit: https://github.com/googleforgames/agones/pull/3529/commits/59900ca5718d9420d48c0402f0240a7dab5e77ac - you had it right in the first place 👍🏻

Reply:
+1 to this, add a note on `priorities` before `template` https://github.com/googleforgames/agones/blob/735e1cf069ffcdf10ff37b2c922e4215da5cf70d/site/content/en/docs/Reference/fleet.md?plain=1#L120 and after `allocationOverflow` to keep with the same ordering as the example yaml.

Reply:
@Kalaiselvi84 just a reminder that this also needs the section at the bottom of the docs here:
https://agones.dev/site/docs/reference/gameserver/#:~:text=The%20spec%20field%20is%20the%20actual%20GameServer%20specification%20and%20it%20is%20composed%20as%20follow%3A